frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -81,19 +88,25 @@
          (fn [e]
            (dom/prevent-default e)
            (dom/stop-propagation e)
-           (set-state (fn [_] {:type :create-theme}))))]
+           (set-state (fn [_] {:type :create-theme}))))
+
+        close-modal
+        (mf/use-fn (fn []
```

Comment:
Is not an error, and can be ignored, but this can be writen with an annonimous fn  #(st/emit...)